data/definitions/items.yml

```diff
@@ -1052,6 +1085,9 @@ clean_guam:
   id: 249
   price: 161
   limit: 10000
+  potion_unf:
+    level: 3
+    make: "guam_potion_unf"
```

Comment:
Left in by mistake?

Reply:
Yes so sorry about that, I am also working on the rest of herblore and i thought i had removed this before pushing, would you like me to remove this?

Reply:
I presumed so, yeah might as well keep it separate